tuner/tuner/libtuner_test.py

```diff
@@ -161,8 +181,18 @@ def test_parse_dispatch_benchmark_results():
         "/mock/base/dir/specs/2_spec.mlir"
     )
 
+    libtuner.save_to_json(parsed_results, path_config.output_json)
+
+    with open(path_config.output_json) as file:
+        json_output = json.load(file)
+
+    assert json_output == expected_json_output
+
 
 def test_parse_model_benchmark_results():
+    path_config = libtuner.PathConfig()
+    object.__setattr__(path_config, "output_json", "output.json")
```

Comment:
Tests shouldn't write files in the current working directory. We can create a temporary directory for tests artifacts if needed.

Reply:
This hasn't been resolved